src/containers/DataGUI.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
Isn't that necessary to export the `class`? Not sure, though.. `Pages` have the same duplication